squad/plugins/gerrit.py

```diff
@@ -106,29 +106,52 @@ def __gerrit_request__(self, build, payload):
         else:
             return Plugin.__gerrit_post__(build, payload)
 
+    def __get_labels__(self, build, success):
+        labels = {}
+        empty = [None, {}]
+        if not success:
+            labels = {'Code-Review': '-1'}
+
+        plugins_settings = build.project.get_setting('plugins')
+        if plugins_settings in empty:
+            return labels
+
+        settings = plugins_settings.get('gerrit')
+        if settings in empty:
+            return labels
+
+        build_finished = settings.get('build_finished')
+        if build_finished in empty:
+            return labels
+
+        if success:
+            labels = build_finished.get('success', {})
```

Comment:
why the difference between success and error?

Reply:
I thought of a scenario where a gerrit patchset could've been flagged -2 on build failure or +1 on build success. Is there such thing or should we just assume +1 for success and -1 for failure?

Reply:
ok, makes sense.